lesson33/index.js

```diff
@@ -0,0 +1,88 @@
+const subjects = {
+    mathematics: {
+        students: 200,
+        teachers: 6
+    },
+    biology: {
+        students: 120,
+        teachers: 6
+    },
+    geography: {
+        students: 60,
+        teachers: 2
+    },
+    chemistry: {
+        students: 100,
+        teachers: 3
+    }
+}
+
+// 1
+function makeString(obj) {
+    const keys = Object.keys(obj)
+    const newString = keys.join(', ')
+    console.log(`All subjects: ${newString}`)
+}
+//  2
+function countStudents(obj) {
+    const arr = Object.keys(obj)
+    let res = 0
+    arr.forEach(key => {
+        let value = obj[key].students
+        res += value
+    })
+    console.log(`Total students: ${res}`)
+}
+
+function countTeachers(obj) {
+    const arr = Object.keys(obj)
```

Comment:
nit: `Object.values`, `Object.entries` can be used instead of `Object.keys` when you work with both keys & values